### PR TITLE
Fix admin checkboxes

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/5366

Admin page -> settings -> users -> admin user -> "edit" -> "roles" tab -> observe the checkboxes

Before:

image

After:

image

### CODE REVIEW

**Review by mmmavis — CHANGES_REQUESTED** [review]
image

check mark becomes almost invisible

File: network-api/networkapi/wagtailcustomization/static/css/admin-interface.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#wagtail .wrapper input[type="checkbox"] {
```
[the remark this answers is not in the document]
Comment (Pomax) [reply network-api/networkapi/wagtailcustomization/static/css/admin-interface.css:1]: good call

**Review by mmmavis — CHANGES_REQUESTED** [review]
the check mark is still pretty invisible to me on Chrome and Firefox 🤔

**Comment by Pomax** [comment]
turns out the CSS rule for "checked" was less specific than the rule for "focussed"... made them equally specific so the checkmark should be nice and thick green again (once heroku updates)

**Review by mmmavis — CHANGES_REQUESTED** [review]
Check mark only shows up for me (super light grey) when the checkbox is on hover.

I commented out all `background` and `color` rules in `admin.css` and it seemed to style the check mark properly. Let's give this a try?

File: network-api/networkapi/wagtailcustomization/static/css/admin.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  color: var(-admin-interface-field-fg);
+}
+
+#wagtail .wrapper li.focused input[type="checkbox"]:before {
+  background: unset;
+}
+
+#wagtail .wrapper li.focused input[type="checkbox"]:not(:checked):before {
+  color: transparent;
+}
```
Comment (mmmavis) [inline network-api/networkapi/wagtailcustomization/static/css/admin.css:28]:
```suggestion
#wagtail .wrapper input[type="checkbox"] {
  vertical-align: middle;
}

#wagtail .wrapper input[type="checkbox"]:before {
  top: calc(50% - 10px);
  left: calc(50% - 10px);
}
```

**Review by mmmavis — CHANGES_REQUESTED** [review]

**Comment by Pomax** [comment]
That works, but restores the weird background color that my CSS was supposed to solve. However, I don't care enough to revisit this, so let's just merge this in and if that background color bugs anyone, they get to fix it themselves.

image

**Review by mmmavis — APPROVED** [review]

**Pull request merged**